src/test/java/org/codeforamerica/shiba/pages/UserJourneyPageTest.java

```diff
@@ -273,7 +201,7 @@ void deletingUploadedFileShouldLoadDocumentUploadScreenUponConfirmDeletion() {
 
     @Test
     void shouldSkipChildcareAssistancePageIfCCAPNotSelected() {
-        completeFlowFromLandingPageThroughReviewInfo(List.of("Food (SNAP)"));
+        completeFlowFromLandingPageThroughReviewInfo(List.of("Food (SNAP)"), smartyStreetClient);
```

Comment:
Having to pass this client into this helper was the only downside of moving helpers into `AbstractBasePageTest`. We can't move the `@MockBean SmartyStreetClient smartyStreetClient` declaration into `AbstractBasePageTest` since some subclasses of `AbstractBasePageTest` require a real SmartyStreetClient.

Really we should probably make an `AbstractUserJourneyPageTest` that extends `AbstractBasePageTest`, but since we might still break up UserJourneyPageTest further, and since we're not quite sure what that will look like, I decided to punt on it.